hal/src/nRF52840/ble_hal.cpp

```diff
@@ -389,7 +389,7 @@ class BleObject::Observer {
     volatile bool isScanning_;                              /**< If it is scanning or not. */
     hal_ble_scan_params_t scanParams_;                      /**< BLE scanning parameters. */
     os_semaphore_t scanSemaphore_;                          /**< Semaphore to wait until the scan procedure completed. */
-    uint8_t scanReportBuff_[BLE_MAX_SCAN_REPORT_BUF_LEN];   /**< Buffer to hold the scanned report data. */
+    uint8_t scanReportBuff_[BLE_MAX_ADV_DATA_LEN_EXT];   /**< Buffer to hold the scanned report data. */
```

Comment:
We should keep `BLE_MAX_SCAN_REPORT_BUF_LEN` as-is and use it for scan buffers.

Reply:
👍